StantUiIosLibrary/Classes/Menus/MainMenu/MenuTabBarDTO.swift

```diff
@@ -32,15 +32,13 @@ public struct MenuTabBarDTO {
             if let data = NSData(contentsOf: nsurl as URL) {
                 if let newImage = UIImage(data: data as Data) {
                     roundedImage = newImage.roundedRectImageFromImage(cornerRadius: 12)
-                    controller.tabBarItem.image = roundedImage.withRenderingMode(.alwaysOriginal)
                 } else {
                     roundedImage = roundedImage.roundedRectImageFromImage(cornerRadius: 12)
-                    controller.tabBarItem.image = roundedImage.withRenderingMode(.alwaysTemplate)
                 }
             } else {
                 roundedImage = roundedImage.roundedRectImageFromImage(cornerRadius: 12)
-                controller.tabBarItem.image = roundedImage.withRenderingMode(.alwaysTemplate)
             }
+            controller.tabBarItem.image = roundedImage.withRenderingMode(.alwaysOriginal)
```

Comment:
I know you only changed this line, but when I saw a lot of if/else was confusing to me to understand the code. If/else always make confusing code hahaha. Here's a little suggestion about how we can remove all of it and make the code more readable:

```suggestion
fileprivate func setRoundedImageToTabBarItem(url: String) {
  var roundedImage = UIColor.lightGray.image()
  roundedImage     = roundedImage.roundedRectImageFromImage(cornerRadius: 12)

    guard let nsurl    = NSURL(string: url)               else { return }
    guard let data     = NSData(contentsOf: nsurl as URL) else { return }
    guard let newImage = UIImage(data: data as Data)      else { return }

    roundedImage = newImage.roundedRectImageFromImage(cornerRadius: 12)

    controller.tabBarItem.image = roundedImage.withRenderingMode(.alwaysOriginal)
}

```

Reply:
Cool suggestion, Chid. I just modified a little bit because in this case I need to return something if enters in guard conditionals.